pandas/core/groupby.py

```diff
@@ -2368,11 +2369,24 @@ def _get_grouper(obj, key=None, axis=0, level=None, sort=True,
     # axis of the object
     if level is not None:
         if not isinstance(group_axis, MultiIndex):
+            # allow level to be a list-like object (e.g., level=[0] or
+            # level=[0,0]) to make this case match the MultiIndex one
```

Comment:
I *think* we accept -1 for levels in other places. e.g. unstack. pls see what we do there (I dont' remember), if so then would be ok.